common/lib/driver-definitions/src/storage.ts

```diff
@@ -291,9 +291,13 @@ export interface IDocumentServiceFactory {
      */
     createDocumentService(resolvedUrl: IResolvedUrl, logger?: ITelemetryBaseLogger): Promise<IDocumentService>;
 
-    // Creates a new document on the host with the provided options. Returns the document service.
+    /**
+     * Creates a new document with the provided options. Returns the document service.
+     * @param createNewSummary - Summary used to create file. If undefined, an empty file will be created and a summary
+     * should be posted later, before connecting to ordering service.
+     */
     createContainer(
-        createNewSummary: ISummaryTree,
+        createNewSummary: ISummaryTree | undefined,
```

Comment:
@wes-carlson this is how it should look correct?

Reply:
yep